arch/inst/C/c.sdsp.yaml

```diff
@@ -13,16 +13,26 @@ definedBy:
   anyOf:
     - C
     - Zca
+    - Zclsd
 base: 64
 assembly: xs2, imm(sp)
 encoding:
-  match: 111-----------10
-  variables:
-    - name: imm
-      location: 9-7|12-10
-      left_shift: 3
-    - name: rs2
-      location: 6-2
+  RV32:
+    match: 111-----------10
+    variables:
+      - name: xs2
+        location: 6-2
```

Comment:
Here, "xs2" must be even as it is part of a register pair, so add the appropriate `not` statement (for the 5 bit value).